src/sass/Fabric.Brand.Icons.Output.scss

```diff
@@ -0,0 +1,82 @@
+// Images Path
+$imagesPath: "https://appsforoffice.microsoft.com/fabric/assets/Product/PNG";
+
+// Icon Names
+$iconList: Access Excel InfoPath Office OneDrive OneNote Outlook PowerPoint Project SharePoint Visio Word;
+
+// Icon Sizes
+$iconSizes: 16 48 96;
+
+// Mixin to loop through iconList and generate classes
+@mixin iconClass($size, $retina) {
+  @each $name in $iconList {
+    &.ms-Brand--#{$name}#{$size} {
+        @include Icon($name, $retina, $size);
+    }
+  }
+}
+
+// Generate icon size classes
+@mixin IconSizeClass() {
+	@each $size in $iconSizes {
+		&.ms-Brand--Icon#{$size} {
+			@include IconSize($size);
+		}
+	}
+}
+
+// Set image container width and height
+@mixin IconSize($size) {
+	width: #{$size}px;
+	height: #{$size}px;
+}
+
+// Set image source and size.
+@mixin Icon($name, $retina, $size) {
+	background-image: url(#{$imagesPath}/#{$name}_#{$size}x#{$retina}.png);
+}
+
+// Default icon set for Pixel Ratio 1
+.ms-Brand {
+	background-size: 100% 100%;
+	@include IconSizeClass();
+	@include iconClass(16, 1);
+	@include iconClass(48, 1);
+	@include iconClass(96, 1);
+}
+
+// Pixel Ratio 1.5
+@media only screen and (min--moz-device-pixel-ratio: 1.5),
```

Comment:
I did a little research and it looks like [`min-resolution` is the standard](https://developer.mozilla.org/en-US/docs/Web/CSS/resolution). For the [widest support](http://caniuse.com/#feat=css-media-resolution) you can use DPI units, where 96dpi is the equivalent of 1x, 144dpi is 1.5x, and so on.

So these multiple media queries with all of the browser prefixes can be replaced with a single:

`@media only screen and (min-resolution: 144dpi)`

Which our autoprefixer process, to make it work on Safari, outputs as:

`@media only screen and (-webkit-min-device-pixel-ratio: 1.5), only screen and (min-resolution: 144dpi)`